scripts/codex/guard.ps1

```
@@ -0,0 +1,443 @@
+  if ($Token.Contains(":")) {
+    $destination = $Token.Split(":", 2)[1]
+  }
+
+  if ([string]::IsNullOrWhiteSpace($destination)) {
+    return $false
+  }
+
+  return $destination -match "^(refs/heads/)?(develop|main|master)$"
+}
+
+function Get-ConfiguredRemotes {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Get-ConfiguredRemotes' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Get-ConfiguredRemotes'

(PSUseSingularNouns)